meta-lmp-base/conf/distro/lmp-mfgtool.conf

```diff
@@ -31,6 +31,5 @@ include conf/machine/include/lmp-mfgtool-partner-custom.inc
 include conf/machine/include/lmp-mfgtool-factory-custom.inc
 
 # Compatibility with other layers
-BBMASK += " \
-    /meta-security/meta-integrity/recipes-kernel/linux/linux-%.bbappend \
-"
+# WARNING: You have included the meta-integrity layer, but 'integrity' has not been enabled in your DISTRO_FEATURES. Some bbappend files and preferred version setting may not take effect. See the meta-integrity README for details on enabling integrity support.
+DISTRO_FEATURES:remove = "integrity"
```

Comment:
We will always have this warning in mfgtools builds right?

Reply:
yes it is true but looking again on that and we can disable it with:
```
INHERIT:remove = "sanity-meta-integrity"
```